addons/l10n_ar/models/account_move.py

```diff
@@ -106,11 +102,6 @@ def _set_afip_service_dates(self):
             if not rec.l10n_ar_afip_service_end:
                 rec.l10n_ar_afip_service_end = rec.invoice_date + relativedelta(day=1, days=-1, months=+1)
 
-    @api.onchange('invoice_date')
```

Comment:
So, you don't want it with the onchange, only on the post when the value has not been set already?  (so, either the user is not even aware, or he has consciously set it himself)  (just confirming)

Reply:
Exactly